### ISSUE TITLE
Godot 4: Visual Script node text box doesn't focus by default

### ISSUE THREAD

**Issue by kreaninw** [godotengine/godot#58161]
### Godot version

v4.0.alpha2.official [79077e6c1]

### System information

Windows 11, Vulkan Clustered, Intel UHD Graphics 620 v30.0.101.1340

### Issue description

When the users click in a text field to edit a node's text box in Visual Script, the focus will not be on the text box. The users will have to click on the text box one more time before they can actually edit the field. This behavior can be seen in the screen recording below:

https://user-images.githubusercontent.com/1403194/154168144-a7a4027d-71e4-42f2-b806-ec8172c08dfe.mp4

This issue is a regression from Godot 3.4. In Godot 3.4, the users do not need to click on the text box as the focus will be on the text box by default. They can edit the text field immediately, as shown in the screen recording below:

https://user-images.githubusercontent.com/1403194/154168382-c36bd99e-573b-48a3-b788-9fbf7998f1e4.mp4

I expect the focus to be on the text box by default since that is the whole reason I go to this field in the first place.

Note: I am not sure whether this is fixed in [merged PR #58026](https://github.com/godotengine/godot/pull/58026) since this issue is happening in a different place altogether.

### Steps to reproduce

1. Add the get() function from the Object base type in VisualScript for example.
2. Try to edit the Str property's text field. The text box popup won't be focused by default.

**Comment by fire** [comment]
@kreaninw I want to thank you for opening these issues for visual script. I'll try to summon more help.

The visual script developer community has been gathering at https://discord.gg/bfR6by5xEs during the various Google Summer of codes and for coordinating visual scripting development.

**Comment by jmb462** [comment]
It's not fixed by #58026 as it only deals with the rename popup.

Must be missing some ``grab_focus()`` elsewhere too.

**Comment by MisoMosiSpy** [comment]
I was able to fix it by adding value_editor[0]->grab_focus(); here, 
https://github.com/godotengine/godot/blob/5d5f80b5295479ca4e12989394b3a2af8819a1ee/editor/property_editor.cpp#L678-L680

However, I think, it may not be a correct/optimal fix as it is outside visual script code.

**Comment by jmb462** [comment]
> I was able to fix it by adding value_editor[0]->grab_focus(); here,
> 
> https://github.com/godotengine/godot/blob/5d5f80b5295479ca4e12989394b3a2af8819a1ee/editor/property_editor.cpp#L678-L680
> 
> However, I think, it may not be a correct/optimal fix as it is outside visual script code.

Yes, but I think it needs a more wide fix. It doesn't concern only this type.
